public/app/features/canvas/runtime/scene.tsx

```diff
@@ -53,6 +54,24 @@ export class Scene {
     this.root = this.load(cfg, enableEditing);
   }
 
+  getNextElementName = (isGroup = false) => {
+    const label = isGroup ? 'Group' : 'Element';
+    let idx = this.byName.size;
```

Comment:
```suggestion
    let idx = this.byName.size +1;
```
Starting at zero looks a bit funny :)